opensrp-chw/build.gradle

```diff
@@ -163,7 +163,7 @@ android {
             buildConfigField "String[]", "ALLOWED_LOCATION_LEVELS_DEBUG", '{"Clinic" , "CHSS" , "CHA"}'
             buildConfigField "String", 'DEFAULT_LOCATION', '"CHA"'
             buildConfigField "String", 'DEFAULT_LOCATION_DEBUG', '"CHA"'
-            buildConfigField "int", "DATABASE_VERSION", '6'
+            buildConfigField "int", "DATABASE_VERSION", '8'
```

Comment:
Is there any reasons why we are skipping db version 7?

Reply:
Same thing here the migration code for 7 existed but was not applied to the database version on the gradle script. After adding the new migration i just bumped it to what was latest.